modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/helper/EndpointHelper.java

```diff
@@ -62,11 +62,10 @@ else if (Objects.equals(
 					pathParameter,
 					HeadlessBuilderConstants.PATH_PARAMETER_ERC)) {
 
-			objectEntry =
-				_objectEntryHelper.getObjectEntry(
-					companyId, ListUtil.fromCollection(relationshipsNames),
-					schema.getMainObjectDefinitionExternalReferenceCode(),
-					pathParameterValue, scopeKey);
+			objectEntry = _objectEntryHelper.getObjectEntry(
```

Comment:
The description of this commit does not match what you did, as you are using the same method. Maybe SF?